src/main/java/seedu/allonus/AllOnUs.java

```diff
@@ -47,7 +118,7 @@ private static void exit() {
      * @return <code>true</code> if is a "goto m/Contacts_Manager" command else <code>false</code>.
      */
     public static boolean isContactsManagerCommand(String userInput) {
-        return userInput.equals("goto m/Contacts_Manager");
+        return areEqual(userInput, CONTACTS_MANAGER_COMMAND);
```

Comment:
Is this change meant to address #116, or is it a code quality improvement?

Reply:
Addresses both. Need to compare userInput and Command type in lower case and wanted to extract a method (since it is common functionality).